consensus/service/src/config/tokens.rs

```diff
@@ -689,25 +712,26 @@ mod tests {
             token_id = 1
             minimum_fee = 1
             [tokens.master_minters]
-            signers = [
-                "302a300506032b65700321000303030303030303030303030303030303030303030303030303030303030303",
-                "302a300506032b65700321007b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b7b"
-            ]
+            signers = """
+            -----BEGIN PUBLIC KEY-----
+            MCowBQYDK2VwAyEAyj6m0NRTlw/R28Q+R7vBakwybuaNFneKrvRVAYNp5WQ=
+            -----END PUBLIC KEY-----
+            -----BEGIN PUBLIC KEY-----
+            MCowBQYDK2VwAyEAl3XVo/DeiTjHn8dYQuEtBjQrEWNQSKpfzw3X9dewSVY=
+            -----END PUBLIC KEY-----
+            """
```

Comment:
Optional/Equivalent: These keys seem to repeat across test cases, so how about we use consts for them? e.g.

```rust
let input = format!(r#"
  ...
  signers="""
    {}
    {}
   """
"#,
  PEM_ENCODED_KEY_1,
  PEM_ENCODED_KEY_2,
);
```

Reply:
My personal opinion is that I think its fairly minimal and easier to read without the string formatting. Part of what I like in having the full TOML/JSON is that it serves as an easy-to-read example of what the file should look like.